src/routes/cards/cards.js

```diff
@@ -146,7 +145,7 @@ export class Cards {
         }
       })
       .catch(response => {
-        console.log(response);
+        console.log(response); //Retain, DO NOT DELETE
```

Comment:
We don't have error catching as of now in case image upload fails for some reason. Logging the error doesn't solve any purpose here, but this piece sits as a placeholder for now. Let's resolve #186 ?